tests/pytests/unit/states/test_kapacitor.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+import salt.states.kapacitor as kapacitor
+from tests.support.mock import Mock, mock_open, patch
+
+
+def _present(
```

Comment:
**suggestion** this should probably be a fixture... I don't understand what's going on with the `isinstance` part though, since nothing appears to pass `define_result` or any of the `_result` in? :confused: